src/style.css

```diff
@@ -1 +1,61 @@
-/* Styles go here */
+* {
+  margin: 0;
+  box-sizing: border-box;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+  /* height: 60px; */
+  box-sizing: border-box;
+  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.05);
+  position: relative;
+}
+
+.nav__list {
+  text-decoration: none;
+  margin-left: 20px;
+  text-transform: uppercase;
+  font-size: 12px;
+  font-weight: 500;
+  color: #000;
+  font-style: normal;
+  font-family: "Roboto", sans-serif;
+  padding-top: 23px;
+  padding-bottom: 23px;
+  line-height: 60px;
+}
+
+.nav__list:hover {
+  color: rgba(0, 172, 220, 1);
+}
+
+.nav__ul {
+  display: flex;
+  list-style-type: none;
+  justify-content: space-between;
+}
+
+.is-active {
+  position: relative;
+  color: rgba(0, 172, 220, 1);
+}
+
+.is-active::after {
+  content: " ";
+  width: 37px;
```

Comment:
And what if other link will be active?

Reply:
<img width="267" alt="Screenshot 2022-08-17 at 18 33 48" src="https://user-images.githubusercontent.com/84275496/185181167-b6006465-08f7-4d84-9f43-705a7c3c933a.png">